tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace/transformation_tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from typing import Any, Optional, Sequence, Union, overload, Literal, Generator
+
+import pytest
+import dace
+import copy
+import numpy as np
+from dace.sdfg import nodes as dace_nodes
+from dace.transformation import dataflow as dace_dataflow
+
+from gt4py.next import common as gtx_common
+from gt4py.next.program_processors.runners.dace_fieldview import (
+    transformations as gtx_transformations,
+)
+
+
+@pytest.fixture(autouse=True)
+def _set_dace_settings() -> Generator[None, None, None]:
```

Comment:
This is a public fixture used everywhere so it should be named accordingly. Also, since this is an important part of the test machinery,  I'd also add a doctring to document which tests are using it and a link to read further documentation about this feature (for other developers reading the code without a deep knowledge of all pytest options).
```suggestion
def set_dace_settings() -> Generator[None, None, None]:
```